z3/include/z3_datatype.h

```diff
@@ -0,0 +1,77 @@
+#pragma once
+#include <memory>
+#include <string>
+#include <vector>
```

Comment:
Are these needed? I don't see them in cvc5_datatype.h.

Reply:
Removed